app/core/tests/test_models.py

```diff
@@ -0,0 +1,35 @@
+from django.test import TestCase
```

Comment:
These tests are ok, but you could also add a test checking for the relation between ingredient and recipe.

Reply:
Added implicit relation on recipe with ingredients creation to check that the link beetwen a recipe and its ingredients is created after recipe creation.